README.md

```diff
@@ -121,11 +121,16 @@ This applies to each `TEMPLATE[X]` section. X is the number of Template starting
 | CUST_POLLING | Polling interval in ms for Device |
 | CUST_Total | Path in JSON where to find total Energy |
 | CUST_Total_Mult | Multiplier to convert W per minute for example in kWh|
+| CUST_Total_Default | [optional] Default value if no value is found in JSON |
```

Comment:
Maybe we should explain the basic idea of this a bit more detailed!? Somewhere else in the readme!?

Reply:
This is still very poorly documented. This is THE topic of this PR, so maybe at least one sentence in the config table: "you could set this to null, so when there is no (valid) value from the device (??) then victron/VRM/venus will display the device as still existant but value is "---"").
Maybe you are able to further improve this sentence...

Reply:
You don't need to set it to any value. Because this PR is working without any config. This is just an optional feature which will override the "---" string to 0, "N/A" or whatever. Of course, I don't know what Victron will do with the overwritten value. 

I don't know what you like to read here. It is optional, it is related to total in this case and replace --- (which is now Default) with a value you set here. 

The PR was that --- was not visible because the code didn't handle an unknown value.